piccolo_admin/endpoints.py

```diff
@@ -262,6 +268,8 @@ def __init__(
         rate_limit_provider: t.Optional[RateLimitProvider] = None,
         production: bool = False,
         site_name: str = "Piccolo Admin",
+        default_language_code: str = "auto",
```

Comment:
It should revert to `default_language_code: str = "en"` because it is English is default language